libtiff/libtiff_ctypes.py

```diff
@@ -519,7 +519,7 @@ def open(cls, filename, mode='r'):
         return tiff
 
     @staticmethod
-    def get_numpy_type(bits, sample_format=None):
+    def get_numpy_type(bits, sample_format=None, byte_swapped=False):
```

Comment:
Would it make sense to use `self.IsByteSwapped()` as a default value to `byte_swapped`?
If so, then eliminate `byte_swapped` argument?